src/clj/fluree/db/database/async.cljc

```diff
@@ -84,30 +83,27 @@
                 (jld-response/-reverse-property iri reverse-spec compact-fn cache fuel-tracker error-ch)
                 (async/pipe prop-ch)))
           (catch* e
-                  (log/error e "Error loading database")
-                  (>! error-ch e))))
+            (log/error e "Error loading database")
+            (>! error-ch e))))
       prop-ch))
 
   (-iri-visible? [_ iri]
     (go-try
       (let [db (<? db-chan)]
         (<? (jld-response/-iri-visible? db iri)))))
 
-
```

Comment:
i think these extra spaces make things more readable because it better separates the different protocol implementations

Reply:
Yeah, I don't necessarily disagree, but the tradeoff is that I don't think we can easily encode that as a rule exception in a tool like cljfmt. And I would rather move towards automatic formatting w/ a tool like that and give up some of these nice formatting exceptions.

Reply:
The other option is that we could disable the default-true `:remove-consecutive-blank-lines?` setting in cljfmt. But we have quite a bit of arbitrary inconsistency here that I would prefer to eliminate as a source of incidental git churn over preserving the places where it can aid in readability.

Reply:
yeah, that makes sense. we can't automate these checks and have them be perfect all the time. at least not yet.